gateway/api/models.py

```diff
@@ -75,13 +75,20 @@ class Program(ExportModelOperationsMixin("program"), models.Model):
         (CIRCUIT, "Circuit"),
     ]
 
+    DEFAULT_DISABLED_MESSAGE = "IBM has temporarily disabled access to this function"
+
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     created = models.DateTimeField(auto_now_add=True, editable=False)
 
     title = models.CharField(max_length=255, db_index=True)
     readable_title = models.CharField(
         max_length=255, null=True, blank=True, default=None
     )
+
+    disabled = models.BooleanField(default=False, null=True)
```

Comment:
The only thing that I would change from this PR is the usage of `null` here. I don't think it has sense to configure it as `True` taking into account that we are setting up a `default` value. So I think `null` should be `False` here.

Reply:
Done in [75ad3a3](https://github.com/Qiskit/qiskit-serverless/pull/1611/commits/75ad3a329950c9c683e49d7d00f86575cd1ac072)